### ISSUE TITLE
Blast results output tables are consistently empty

### ISSUE THREAD

**Issue by Sveta-user** [PATRIC3/patric3_website#2436]
The new great Blast service at BV-BRC generates a downloadable output CSC or text  table, but it is empty.  I checked any times
![empty_blast_output_BV-BRC](https://user-images.githubusercontent.com/20665963/189731721-705153b2-2a17-4ddf-96c4-856054fd5a07.png)

**Comment by olsonanl** [comment]
I think the download there will download the selected rows - try checking the checkbox just to the left of where it says "Query ID"

**Comment by Sveta-user** [comment]
Just try selecting "all" rows, selecting just a few, or NO selection - all 3 downloaded files came back empty
[BVBRC_blast (4).csv](https://github.com/PATRIC3/patric3_website/files/9550964/BVBRC_blast.4.csv)

**Comment by olsonanl** [comment]
OK, we'll need to check into it. That same data is available in the job folder, in the file called blast_out.txt.

**Comment by Sveta-user** [comment]
True!  But from there these files are not downloadable either for some reason .  weird..  The is not an urgent or crucial issue.  I just need the output files for recordkeeping, but as long as they are safely kept in BRC - I am ok ;))
[BVBRC_blast.txt](https://github.com/PATRIC3/patric3_website/files/9551017/BVBRC_blast.txt)

**Comment by olsonanl** [comment]
What is the job number for that job?

From: Sveta-user ***@***.***>
Date: Monday, September 12, 2022 at 2:01 PM
To: PATRIC3/patric3_website ***@***.***>
Cc: Olson, Robert D. ***@***.***>, Comment ***@***.***>
Subject: Re: [PATRIC3/patric3_website] Blast results output tables are consistently empty (Issue #2436)

True! But from there these files are not downloadable either for some reason . weird.. The is not an urgent or crucial issue. I just need the output files for recordkeeping, but as long as they are safely kept in BRC - I am ok ;))
BVBRC_blast.txt<https://github.com/PATRIC3/patric3_website/files/9551017/BVBRC_blast.txt>

**Comment by Sveta-user** [comment]
<html>
<body>
<h3 class="normal-case close2x" style="margin: 0px 0px 0.4em; padding: 0px; font-size: 1.3em; font-weight: normal; line-height: 1; border: 0px; font-family: Arial, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; color: rgb(136, 136, 136);"><span class="wrap" style="margin: 0px; padding: 0px; border: 0px;">Homology Job Result</span></h3>

Job ID | 9486259
-- | --
9/12/22, 1:11 PM
9/12/22, 1:13 PM
2m16s


</body>
</html>

**Comment by olsonanl** [comment]
Hm, I was able to look at your job and successfully download the output. Clicked on the filename, then clicked "DWNLD" in the green bar.

**Comment by Sveta-user** [comment]
Hmm...  Then this is probably a security issue on the IFF side... unfortunately!  I will talk to IT people on my side to allow BRC full access for me or  smth.  But they are strict  here, hmmm... We try at least

**Comment by Sveta-user** [comment]
I assume the BRC site has the current license and other formalities, right?

**Comment by olsonanl** [comment]
Yes, same policies as PATRIC

**Comment by Sveta-user** [comment]
Hi Bob,
This is not urgent or crucial - reporting just to be a good citizen ;) 
I have tried downloading Blast output Tables from my home Mac (bypassing IFF security ) .  And I also tried both, Chrome and Firefox browsers.   So - I can download the simple file type from my Jobs: blast_out.txt (attached).  So I am OK.  But the new VB-BRC's  Blast output format is nicer (as it includes full genome names + even Query seq info).   Those Blast outputs still come out empty for me - both, in a CSV or txt formats  
<img width="607" alt="image" src="https://user-images.githubusercontent.com/20665963/190489879-03e3e37b-3ea4-45a9-b53b-ecb744bacd3f.png">


![BRC_download](https://user-images.githubusercontent.com/20665963/190490599-ee077ea4-c597-4080-8cc9-671b66b856db.png)

**Comment by anddomen** [comment]
For what it's worth, I also have this problem. No matter what box is checked, both the CSV and text come out empty when downloaded.
